src/docs/release-notes/R24/FullNotes.mdx

```diff
@@ -0,0 +1,31 @@
+### Documentation Site
+
+-   Added an Announcement Banner to the documentation homepage.
```

Comment:
This thing has been around for a while. We only recently cleaned up the code and made it read data from blui-database. For our users, this change is not substantial. Consider removing it or just say "performance improvement"

Reply:
I've removed the announcement banner.